### PR TITLE
Create ci deployment

### PR DESCRIPTION
**Pull request by kuronosec**
add new github workflow files for the monitor deployment in k8s. refs #86

### CODE REVIEW

**Comment by kuronosec** [comment]
Ups I think I merged as well the changes that were pushed to master but not to dev.

File: .env.example [hunk 1]
```diff
@@ -18,7 +18,7 @@ HASURA_GRAPHQL_ADMIN_SECRET=myadminsecretkey
 HASURA_GRAPHQL_UNAUTHORIZED_ROLE=guest
 
 # hapi
-HAPI_EOS_API_ENDPOINT=https://jungle2.eosio.cr
+HAPI_EOS_API_ENDPOINT=https://jungle3.cryptolions.io
```
Comment (xavier506) [inline .env.example:21]: we can use our jungle endpoint here https://jungle.eosio.cr (this is jungle 3)

File: .env.example [hunk 2]
```diff
@@ -36,7 +39,7 @@ REACT_APP_EOS_RATE_LINK=https://jungle.eosrate.io:8080
 REACT_APP_USE_REWARDS=true
 REACT_APP_USE_VOTES=true
 REACT_APP_USE_BLOCK_PRODUCER_AGREEMENT_CONTRACT=true
-REACT_APP_EOS_API_HOST=jungle.eosio.cr
+REACT_APP_EOS_API_HOST=jungle3.cryptolions.io
```
Comment (xavier506) [inline .env.example:42]: https://jungle.eosio.cr

File: .env.example [hunk 3]
```diff
@@ -47,3 +50,4 @@ REACT_APP_EOS_BP_JSON_ON_CHAIN_CONTRACT=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_TABLE=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
 REACT_APP_EOS_ENDPOINT=https://jungle.eosio.cr
```
Comment (xavier506) [inline .env.example:52]: 👍

File: .env.jungle [hunk 4]
```diff
@@ -18,7 +18,7 @@ HASURA_GRAPHQL_ADMIN_SECRET=myadminsecretkey
 HASURA_GRAPHQL_UNAUTHORIZED_ROLE=guest
 
 # hapi
-HAPI_EOS_API_ENDPOINT=https://jungle2.eosio.cr
+HAPI_EOS_API_ENDPOINT=https://jungle3.cryptolions.io
```
Comment (xavier506) [inline .env.jungle:21]: https://jungle.eosio.cr

**Comment by xavier506** [comment]
there seems to be lots of files changed is this because we are updating dev branch from master?

**Comment by kuronosec** [comment]
@xavier506 yeah: https://github.com/eoscostarica/eosio-monitor/pull/96#issuecomment-677769194 I made that mistake. Master was being used by default and dev was behind. We need to update dev anyway.

**Pull request merged**